cmd/internal/printer/printer.go

```
@@ -61,16 +78,52 @@ func (p Config) node(t *gen.Type) {
 				return name == "Name" && i == 0 || name == header[i]
 			})
 			row[i] = fmt.Sprint(field.Interface())
+			_, err := strconv.Atoi(row[i])
+			if err == nil {
+				alignment = append(alignment, tw.AlignRight)
+			} else {
+				alignment = append(alignment, tw.AlignLeft)
+			}
 		}
 		row[len(row)-1] = f.Comment()
-		table.Append(row)
+		err := table.Append(row)
+		if err != nil {
+			return
+		}
```

In case there is an error, it should not be swallowed.